tidy3d/components/near2far.py

```diff
@@ -297,7 +286,7 @@ def _resample_surface_currents(
         currents: xr.Dataset,
         sim_data: SimulationData,
         surface: Near2FarSurface,
-        medium: Medium,
+        medium: MediumType,
```

Comment:
can it be any type of medium? or just `Medium` (dispersionless?)

Reply:
The default projection distance is now `1m`, and I think it's fair game to project through a lossy material at a finite distance. The radar cross section specifically is an `r -> \infty` quantity, so for that there's a specific check to make sure that the material is not lossy. Maybe I should add a similar check for the diffraction monitor, since that also leads to fields at `r -> \infty`.

Reply:
Yeah that would make sense to me, adding it also to diffraction monitor.